### PR TITLE
#15

### PR DESCRIPTION
**Pull request by maehr**
As discussed in #15 I tried to merge our lists. Please feel free to add and/or modify my changes.

### CODE REVIEW

**Comment by ashwinvis** [comment]
I have will make some minor modifications and merge it soon.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```
Comment (ashwinvis) [inline LICENSE.md:1]: A LICENSE file was already present and this is not exactly markdown per se. Is this intentional or accidental?

File: README.md [hunk 2]
```diff
@@ -26,7 +26,56 @@ document**.<br/>
...
+- [R Studio](https://github.com/rstudio/rstudio) - IDE for R.
+  - [bookdown](https://github.com/rstudio/bookdown) - R package to facilitate writing books and long-form articles, reports with R Markdown :bookmark: :link:.
+  - [R Markdown](https://rmarkdown.rstudio.com/) - R package to write R next to Markdown
+   :bookmark:
+   :link:.
+- [Vim](https://www.vim.org/) - Command line text editor.
+- [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
+ with vim integration which uses fzf (a fuzzy finder implemented in Go).
+  -[vim-pandoc](https://github.com/vim-pandoc/vim-pandoc) - Pandoc integration and utilities for Vim.
+  -[vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax) - Pandoc syntax highlighting for Vim.
```
Comment (ashwinvis) [inline README.md:44]: 1. fzf-bibtex is not tied to Vim but let's keep it this way unless somebody complains :smile: 
2. Fix indentation:
```suggestion
  - [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
 with vim integration which uses fzf (a fuzzy finder implemented in Go).
  - [vim-pandoc](https://github.com/vim-pandoc/vim-pandoc) - Pandoc integration and utilities for Vim.
  - [vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax) - Pandoc syntax highlighting for Vim.
```

File: README.md [hunk 3]
```diff
@@ -26,7 +26,56 @@ document**.<br/>
...
+Reference managers to generate citations, BibTeX, and BibLaTeX files.
+
+- [Citation Style Language (CSL) styles](https://editor.citationstyles.org/) -  A
+ crowdsourced repository with over 9000 free CSL citation styles and an online
+ editor to create new ones.
+- [JabRef](https://www.jabref.org/) - Open source bibliography reference manager.
+- [Zotero](https://www.zotero.org/) - FOSS tool to collect, organize, cite, and
+ share research.
+  -[Better BibTeX for Zotero](https://retorque.re/zotero-better-bibtex/) - Enhanced BibLaTex integration for Zotero.
+  -[ZotFile for Zotero](http://zotfile.com/) - Enhanced PDF file management for Zotero.
```
Comment (ashwinvis) [inline README.md:61]:
```suggestion
  - [Better BibTeX for Zotero](https://retorque.re/zotero-better-bibtex/) - Enhanced BibLaTex integration for Zotero.
  - [ZotFile for Zotero](http://zotfile.com/) - Enhanced PDF file management for Zotero.
```

File: README.md [hunk 4]
```diff
@@ -53,60 +99,25 @@ Word Processors which can generate LaTeX, HTML or PDF output on demand.
...
+## Spell Checking and Linting
 
+- [GNU Aspell](http://aspell.net/)
+- [Hunspell](http://hunspell.github.io/)
 - [LanguageTool](https://languagetool.org/) - Open source grammar, style and
  spell Checker.
+- [Markdown lint tool](https://github.com/markdownlint/markdownlint)
 - [proselint](http://proselint.com/) - A linter for prose.
+- [remarklint](https://github.com/remarkjs/remark-lint)
+- [restructuredtext-lint](https://github.com/twolfson/restructuredtext-lint)
```
Comment (ashwinvis) [inline README.md:120]: Add a small description per tool as you see fit.

File: README.md [hunk 5]
```diff
@@ -26,7 +26,56 @@ document**.<br/>
...
+  - [bookdown](https://github.com/rstudio/bookdown) - R package to facilitate writing books and long-form articles, reports with R Markdown :bookmark: :link:.
+  - [R Markdown](https://rmarkdown.rstudio.com/) - R package to write R next to Markdown
+   :bookmark:
+   :link:.
+- [Vim](https://www.vim.org/) - Command line text editor.
+- [fzf-bibtex](https://github.com/msprev/fzf-bibtex/#readme) - A BibTeX source
+ with vim integration which uses fzf (a fuzzy finder implemented in Go).
+  -[vim-pandoc](https://github.com/vim-pandoc/vim-pandoc) - Pandoc integration and utilities for Vim.
+  -[vim-pandoc-syntax](https://github.com/vim-pandoc/vim-pandoc-syntax) - Pandoc syntax highlighting for Vim.
+- [Visual Studio Code](https://code.visualstudio.com/) - Popular IDE with Markdown support.
```
Comment (ashwinvis) [inline README.md:45]:
```suggestion
- [Visual Studio Code](https://code.visualstudio.com/) - Popular IDE with Markdown support.
  - [Markdown All in One](https://github.com/yzhang-gh/vscode-markdown/#readme) - Extension for enhanced 
    markdown support in VSCode, such as preview and auto completion to name a few.
```

File: LICENSE.md (hunk 1 above)
Comment (ashwinvis) [reply LICENSE.md:1]: The main guidelines also suggest:

> Place a file named license or LICENSE in the repo root with the license text.

Comment (maehr) [reply LICENSE.md:1]: Pardon me. I will remove the file.

File: README.md (hunk 4 above)
Comment (maehr) [reply README.md:122]: I will add a description asap.

**Comment by maehr** [comment]
@ashwinvis The removal of the white spaces suggested by you make awesome-lint unhappy. I don't know if it is a bug in awesome-lint or intentionally set like this.

**Comment by ashwinvis** [comment]
No clue... guess one has to run it locally. Do you have npm / yarn installed? It should be something like:

```sh
npm install -g awesome-lint
awesome-lint README.md
```

**Comment by ashwinvis** [comment]
Have a look at how to disable certain rules: https://github.com/sindresorhus/awesome-lint#special-comments

**Comment by ashwinvis** [comment]
I figured it out. It is a bug in awesome-lint which is too strict about markdown format. And there is a silly workaround.

**Comment by ashwinvis** [comment]
Obsoleted by #21

**Pull request closed without merging**